style.css

```diff
@@ -484,7 +488,363 @@ footer {
 .liner {
   height: 5px;
   width: 134px;
-  margin: 13px 121px 0 120px;
+  margin-top: 13px;
+  margin-left: 125px;
   background-color: #0d151e;
   border-radius: 10px;
 }
+
+/* -------------------------------------------------------
+ -------desktop version style using media query--------
+------------------------------------------------- */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
+
+  /* header/ navbar media query start */
+
+  .navbar-container {
+    min-height: 72px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .nav-ul {
+    width: 50%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  .nav-ul li a {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: var(--color-white);
+  }
+
+  /* header/ navbar media query end */
+
+  /* headline/homepage media query star */
+
+  .section {
+    min-height: 500px;
+    background-color: var(--color-black);
+    background-image: url("./images/desktop-1.svg");
+    background-repeat: no-repeat;
+    background-size: 50%;
+    background-position-x: 100%;
+    background-position-y: 90%;
+  }
+
+  .headline-container {
+    width: 50%;
+    flex-direction: row-reverse;
+  }
+
+  .headline-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 85%;
+    height: 100%;
+    padding-right: 10%;
+  }
+
+  .headline-text h1 {
+    font-size: 50px;
+  }
+
+  .headline-text h3 {
+    font-size: 25px;
+  }
+
+  .social-media-icons {
+    width: 15%;
+    margin-top: 25%;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  /* work section media query for desktop version */
+
+  .works-container {
+    width: 85%;
+    min-height: 800px;
+    margin: 1px 10%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-title-dv {
+    width: 100%;
+    height: 100%;
+    margin-bottom: 0;
+    display: flex;
+    align-items: center;
+  }
+
+  .works-title {
+    font-size: 16px;
+    line-height: 20px;
+    margin: 2px;
+    margin-top: 55px;
+    width: 20%;
+  }
```

Comment:
- **[OPTIONAL]** Kindly adjust the font styles to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/214888403-d9a00ce2-ddf7-4e9e-9597-866ade83c401.png) | ![image](https://user-images.githubusercontent.com/58642949/214888202-c1619bc2-7e99-41c2-af0c-ab4b8065142a.png) |